### DISCUSSION TITLE
Works for Sequoia?

### DISCUSSION THREAD

**Discussion by verajosemanuel** [godbout/kindaVim.blahblah#261] [Q&A]
Download page just offers a button for Ventura.

Is the app updated for latest macOS?

Regards

**Comment by godbout** [comment]
short answer: **the latest kV (kV69 at time of writing) runs on Ventura, Sonoma and Sequoia**.

less short answer:
<img width="553" alt="Screenshot 2024-12-31 at 14 34 38" src="https://github.com/user-attachments/assets/c255d361-0c75-481e-96bd-7373c10e866d" />

Ventura<b>++</b> (++) means Ventura and above. Ventura is the *minimum* version of macOS required in order to run the latest kV. kV [follows Apple and only supports the latest three macOS releases](https://endoflife.date/macos), so that the code can be clean, using all the newest APIs, and be as efficient as possible blah blah blah. you can see the list of releases of kV here: https://github.com/godbout/kindaVim.docs/releases. should have an average of two releases a month over the past four years.

for those who can't upgrade to the latest macOS versions, they'll always be able to download a past version of kV for their relevant macOS versions as you can see in the screenshot. those who currently are stuck with Monterey can still download and use kV66.

**Comment by godbout** [comment]
if this is not clear to you, or you find the button on the site confusing lemme know. although it's pretty standard. you usually write the lowest version available on a download page. like macOS 12.3+ etc. (although in my case i prefer using the release names rather than the numbers. numbers seem more appropriate for computers, names for humans). if this is clear then we can close. thanks.

**Reply by verajosemanuel** [reply]
that is ok, somewhat assumed the ++ as the numbering for the version (sometimes I am so dumb...) Thanks

**Reply by godbout** [reply]
no worries. definitely not your fault. i just like to do things my way, and the right way should have been to write `Download for macOS 13.5+`.  still can't get to understand this right away, and it looks ugly. so i'm sticking with my `Ventura++`. my rule as long as i'm not complaining about the lack of users or subscribers i'm allowed to do the fuck i want hahahahahahahhha. cheers.
